tests/wp-unit/include/Core/Posts/Modules/DeletePostsByTaxonomyModuleTest.php

```diff
@@ -26,6 +26,9 @@ public function setUp() {
 		$this->module = new DeletePostsByTaxonomyModule();
 	}
 
+	/**
+	 * Test deleting posts from a single taxonomy term default post type.
+	 */
 	public function test_deleting_posts_from_built_in_taxonomy_terms() {
```

Comment:
When writing comments, start the first letter in Caps.

https://github.com/sudar/bulk-delete/blob/a239562313b008b47c6681fc3ff4c34fb305ed02/tests/wp-unit/include/Core/Posts/Modules/DeletePostsByTaxonomyModuleTest.php#L71